docs/edgeql/transactions.rst

```diff
@@ -47,18 +66,35 @@ Using an EdgeQL query string:
 .. code-block:: typescript
 
   client.transaction(async tx => {
-    await tx.execute(`insert Fish { name := 'Wanda' };`);
+    await tx.execute(`update BankCustomer 
+      filter .name = 'Customer1'
+      set { bank_balance := .bank_balance -10 }`);
+    await tx.execute(`update BankCustomer 
+      filter .name = 'Customer2'
+      set { bank_balance := .bank_balance +10 }`);
   });
 
 Using the querybuilder:
 
 .. code-block:: typescript
 
-  const query = e.insert(e.Fish, {
-    name: 'Wanda'
+  const query1 = e.update(e.BankCustomer, {
+    filter_single: { name: "Customer1" },
+    set: {
+      bank_balance: { "-=":  10 }
+    },
   });
-  client.transaction(async tx => {
-    await query.run(tx);
+
+  const query2 = e.update(e.BankCustomer, {
+    filter_single: { name: "Customer2" },
+    set: {
+      bank_balance: { "+=":  10 }
+    },
+  });
```

Comment:
Oops! I forgot that `e.update` introduces a scope: another small update:

```suggestion
  const query1 = e.update(e.BankCustomer, () => ({
    filter_single: { name: "Customer1" },
    set: {
      bank_balance: { "-=":  10 }
    },
  }));
  const query2 = e.update(e.BankCustomer, () => ({
    filter_single: { name: "Customer2" },
    set: {
      bank_balance: { "+=":  10 }
    },
  }));
```

Reply:
@scotttrinh Odd, getting a "Applying suggestions on deleted lines is currently not supported." and can't commit this as you suggested. Did it manually here instead: https://github.com/edgedb/edgedb/pull/5780/commits/6582508c10c80591048ddc976a108fd7401b06fc Thanks!

Reply:
Ah, sure, I actually just manually typed `tick-tick-ticksuggestion` because it didn't give me the option for probably the same reason 😅